src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -34,11 +34,12 @@
               type="button"
               @click="clearQuery"
             >
-              <CloseIcon />
+              <CloseIcon decorative />
             </button>
             <SearchIcon
               v-else
               class="code-block-search-icon"
+              decorative
```

Comment:
question: since there is no text around this icon, wouldn't the title/aria-description be needed?

Reply:
It doesn't look like the button wraps the search icon, just the close icon...?

Reply:
Oh yeah sorry - there is `aria-label` on input field itself - this icon really is decorative